extensions/cornerstone/src/hooks/useActiveViewportSegmentationRepresentations.ts

```diff
@@ -0,0 +1,219 @@
+import { useState, useEffect } from 'react';
+import debounce from 'lodash.debounce';
+import { roundNumber } from '@ohif/core/src/utils';
+import {
+  SegmentationData,
+  SegmentationRepresentation,
+} from '../services/SegmentationService/SegmentationService';
+
+const excludedModalities = ['SM', 'OT', 'DOC', 'ECG'];
```

Comment:
Can you define this somewhere else so we use a consistent list?  It seems to vary all over the place - no idea why OT is excluded.

Reply:
OT Excluded for now